tpp-app/src/settings/Settings.js

```diff
@@ -78,14 +83,44 @@ const Preferences = (props) => {
 
 }
 
-
 export default function Settings () {
     const [cycleLength, setCycleLength] = useState(35);
     const [periodLength, setPeriodLength] = useState(5);
     const [remindPeriodEnabled, setRemindPeriodEnabled] = useState(true);
+    const [remindSymptomsEnabled, setRemindSymptomsEnabled] = useState(false);
+
     const togglePeriodSwitch = () => setRemindPeriodEnabled(!remindPeriodEnabled);
-    const [remindSymptomsEnabled, setRemindSymptomsEnabled] = useState(true);
-    const toggleSymptomsSwitch = () => setRemindSymptomsEnabled(!remindSymptomsEnabled);
+    const toggleSymptomsSwitch = () => {
+
+        setRemindSymptomsEnabled(!remindSymptomsEnabled);    
+        
+        if (remindPeriodEnabled) {
+            // Schedule a reoccuring notification 
+            PushNotificationIOS.addNotificationRequest({
+                id: 'remindsymptoms',
+                title: 'Daily Log Reminder',
+                body: 'Daily reminder to log your symptoms!',
+                badge: 1,
+//                fireDate: getCorrectDate(),
+                repeats: true,
+                repeatsComponent: {
+                    hour: true,
+                    minute: true,
+                },
+            });
+        } else {
+            PushNotificationIOS.removePendingNotificationRequests(['remindsymptoms'])
+        }
+    }
+
+    const getCorrectDate = () => {
```

Comment:
Not sure if the comment out on the line `fireDate: getCorrectDate()` is on purpose. If it is, you may as well delete the commented-out line, as well as `getCorrectDate`.